src/proposals/20230830/SparkEthereum_20230830.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import { SparkPayloadEthereum, IEngine, Rates, EngineFlags } from '../../SparkPayloadEthereum.sol';
+
+import { IPoolAddressesProvider } from 'aave-v3-core/contracts/interfaces/IPoolAddressesProvider.sol';
+
+/**
+ * @title  August 30, 2023 Spark Ethereum Proposal - Update ETH Kink, Update ETHn Kink Rate, Update wstETH Supply Cap
+ * @author Phoenix Labs
+ * @dev    This proposal updates ETH market parameters and raises wstETH supply cap
```

Comment:
Describe specific param changes here: variableRateSlope1, variableRateSlope2, optimalUsageRation, and supplyCap for wstETH